scrapers/de/bills.py

```diff
@@ -276,8 +300,13 @@ def scrape_vote(self, bill, vote_id, session):
 
             yield vote
 
-    def add_sponsor_by_legislator_id(self, bill, legislator_id, sponsor_type):
-        sponsor = self.legislators[str(legislator_id)]
+    def add_sponsor_by_legislator_id(
+        self, bill, legislator_id, sponsor_type, sponsor_key="PersonId"
```

Comment:
Adding `sponsor_key` with a default value preserves the old functionality pattern.